src/platform/device-manager.ts

```diff
@@ -4,22 +4,21 @@
  **/
 /// <reference types="@google/local-home-sdk" />
 
-import { DeviceManager } from "@google/local-home-sdk";
-
-export class DeviceManagerStub implements DeviceManager{
+import { DeviceManager } from '@google/local-home-sdk';
 
+export class DeviceManagerStub implements DeviceManager {
   markPending(request: smarthome.IntentRequest): Promise<void> {
     //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
+    throw new Error('Method not implemented.');
   }
 
   getProxyInfo(id: string): smarthome.ProxyInfo {
     //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
+    throw new Error('Method not implemented.');
   }
 
-  deviceId: string;
-  error: boolean;
+  deviceId: string = '';
+  error: boolean = true;
```

Comment:
We default the error to being true?

Reply:
Deleted this variable. Going to implement in #23.